application/src/main/java/org/togetherjava/tjbot/features/utils/LinkFilter.java

```diff
@@ -0,0 +1,17 @@
+package org.togetherjava.tjbot.features.utils;
+
+/**
+ * Possible ways to filter a link.
+ * 
+ * @see LinkDetections
+ */
+public enum LinkFilter {
```

Comment:
cool cool. btw, it would have been okay for me if u added it as "nested" to the other class 👍

Reply:
I wasn't sure lmao
ok going to fix it